tests/chains/THORChain/TWSwapTests.cpp

```diff
@@ -54,7 +56,7 @@ TEST(TWTHORChainSwap, SwapBtcToEth) {
 
     // serialize input
     const auto inputData_ = input.SerializeAsString();
-    EXPECT_EQ(hex(inputData_), "0801122a62633171706a756c7433346b3973706a66796d38687373326a72776a676630786a6634307a65307070381a0708021203455448222a3078623966353737316332373636346266323238326439386530396437663530636563376362303161372a2a62633171366d397532717375386d68387937763872723279776176746a38673561727a6c796863656a373a07313030303030304212313430303030303030303030303030303030");
+    EXPECT_EQ(hex(inputData_), "0a020801122a62633171706a756c7433346b3973706a66796d38687373326a72776a676630786a6634307a65307070381a0708021203455448222a3078623966353737316332373636346266323238326439386530396437663530636563376362303161372a2a62633171366d397532717375386d68387937763872723279776176746a38673561727a6c796863656a373a07313030303030304212313430303030303030303030303030303030");
```

Comment:
A don't understand how the serialized input has changed here (probably due to the change in proto), but they have not changed in `SwapTests.cpp`.

Reply:
It's because in TWSwap we are serializing `SwapInput` - in swap test we serialize the underlying chain